chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -8,11 +8,21 @@ option-or-alt =
         [macos] { general-key-option }
         *[other] { general-key-alt }
     }
+command-or-control =
+    { PLATFORM() ->
+        [macos] { general-key-command }
+        *[other] { general-key-control }
+    }
 return-or-enter =
     { PLATFORM() ->
         [macos] Return
         *[other] Enter
     }
+backsace-or-delete =
```

Comment:
→ `delete-or-backspace` (and note the typo)

Reply:
Oh, man... I really need to start re-reading everything I type two or three times. Sorry about that!